web/src/components/navbar/NavBar.test.tsx

```diff
@@ -240,6 +246,7 @@ describe("<NavBar />", () => {
       expect(menuEl).toBeInTheDocument();
       expect(screen.queryByText(Config.navigationDefaults.logoutButton)).not.toBeInTheDocument();
       expect(screen.queryByText(Config.navigationDefaults.myNJAccountText)).not.toBeInTheDocument();
+      expect(screen.queryByText(Config.navigationDefaults.addBusinessButton)).not.toBeInTheDocument();
     });
 
     it("displays an open dropdown menu when clicked and closes when clicked again", async () => {
```

Comment:
[sand] I noticed the description mentions the drop-down menu is closed when it's clicked for the second time, it looks like the click action to close the menu doesn't happen in this test